libimage/copier.go

```diff
@@ -325,14 +329,22 @@ func (r *Runtime) newCopier(options *CopyOptions) (*copier, error) {
 	return &c, nil
 }
 
-// close open resources.
-func (c *copier) close() error {
+// newCopier creates a copier.  Note that fields in options *may* overwrite the
+// counterparts of the specified system context.  Please make sure to call
+// `(*copier).close()`.
+func (r *Runtime) newCopier(options *CopyOptions) (*copier, error) {
```

Comment:
(Absolutely non-blocking: having a method of `Runtime` in the middle of the implementation of `copier` is a tiny bit untidy. Consider moving it before the `NewCopier` constructor, or to the end of the file.)